backend/posts/serializers.py

```diff
@@ -10,3 +10,12 @@ class Meta:
         model = Post
         fields = ('author', 'title', 'text')
         read_only_fields = ('author',)
+
+class CommentSerializer(serializers.ModelSerializer):
+    author = CustomUserSerializer(read_only=True)
+    post = PostSerializer(read_only=True)
+
+    class Meta:
+        model = Comment
+        fields = ('author', 'post', 'text')
+        read_only_fields = ('author', 'post', )
```

Comment:
remove this read_only_fields because it is reduntant, the same for PostSerializer